cypress/e2e/edge-application/instantiate-function-rules-engine.cy.js

```diff
@@ -1,138 +0,0 @@
-import generateUniqueName from '../../support/utils'
-import selectors from '../../support/selectors'
-
-let fixtures = {}
-
-/**
- * Creates a new edge function.
- */
-const createFunctionCase = () => {
-  cy.openProduct('Edge Functions')
-
-  // Act
-  cy.get(selectors.functions.createButton).click()
-  cy.get(selectors.functions.nameInput).clear()
-  cy.get(selectors.functions.nameInput).type(fixtures.functionName, { delay: 0 })
-  cy.get(selectors.functions.saveButton).click()
-  cy.verifyToast('success', 'Your edge function has been created')
-}
-
-/**
- * Creates a new edge application with basic settings.
- */
-const createEdgeApplicationCase = () => {
-  // Act
-  cy.get(selectors.edgeApplication.mainSettings.createButton).click()
-  cy.get(selectors.edgeApplication.mainSettings.nameInput).type(fixtures.edgeApplicationName)
-  cy.get(selectors.edgeApplication.mainSettings.addressInput).clear()
-  cy.get(selectors.edgeApplication.mainSettings.addressInput).type('httpbingo.org')
-  cy.get(selectors.form.actionsSubmitButton).click()
-  cy.verifyToast('success', 'Your edge application has been created')
-  cy.get(selectors.form.actionsCancelButton).click()
-
-  // Assert - Verify the edge application was created
-  cy.get(selectors.list.searchInput).type(`${fixtures.edgeApplicationName}{enter}`)
-  cy.get(selectors.list.filteredRow.column('name')).should(
-    'have.text',
-    fixtures.edgeApplicationName
-  )
-
-  // Act - Navigate to the created edge application
-  cy.get(selectors.list.filteredRow.column('name')).click()
-}
-
-describe('Edge Application', { tags: ['@dev4', '@xfail'] }, () => {
-  beforeEach(() => {
-    fixtures.edgeApplicationName = generateUniqueName('EdgeApp')
-    // Login
-    cy.login()
-
-    fixtures = {
-      functionName: generateUniqueName('EdgeFunction'),
-      functionInstanceName: generateUniqueName('FunctionsInstance'),
-      edgeApplicationName: generateUniqueName('EdgeApp'),
-      originName: generateUniqueName('origin'),
-      rulesEngineName: generateUniqueName('RulesEng'),
-      cacheSettingName: generateUniqueName('cacheSetting')
-    }
-  })
-
-  it('should instantiate a function in a rules engine', () => {
```

Comment:
This test scenario was moved somewhere else?

Reply:
a gente ja tem outro teste que cobre esse cenário, esse teste está mais completo e mais atualizado: https://github.com/aziontech/azion-console-kit/blob/dev/cypress/e2e/edge-application/rules-engine/create-rule-engine-set-function.cy.js